test/membrane/error.spec.js

```diff
@@ -97,11 +99,14 @@ it('unhandled promise rejections with non-error objects', (done) => {
 })
 
 it('unhandled promise rejections with non-error objects handled in blue', (done) => {
-    window.addEventListener("unhandledrejection", event => {
+    function handler(event) {
         expect(event.reason.foo === undefined).toBe(true)
         expect(event.reason.message === undefined).toBe(true)
+        window.removeEventListener('unhandledrejection', handler);
         done()
-    });
+    }
+
```

Comment:
newline nit

Reply:
I wanted this newline. I added one in the previous test too, for better readability